app/models/organization.rb

```diff
@@ -7,7 +7,14 @@ class Organization < ApplicationRecord
   INSTITUTE = 'http://vivoweb.org/ontology/core#Institute'
   SCHOOL = 'http://vivoweb.org/ontology/core#School'
   UNIVERSITY = 'http://vivoweb.org/ontology/core#University'
-  store_accessor :metadata, :type, :country
-  scope :departments, -> { where("metadata->>'type' = ?", DEPARTMENT).order('name') }
+  store_accessor :metadata, :type, :parent_school
+  # scope :departments, -> { where("metadata->>'type' = ?", DEPARTMENT).order('name') }
   scope :schools, -> { where("metadata->>'type' = ?", SCHOOL).order('name') }
+  def self.departments(parent_school: nil)
+    if parent_school
```

Comment:
how about this instead? I think this betters shows that the conditional only adds a single extra clause.
```ruby
scope = where("metadata->>'type' = ?", DEPARTMENT)
scope = scope.where("metadata->>'parent_school' = ?", parent_school) if parent_school
scope.order('name')
```